build.gradle

```diff
@@ -451,8 +451,11 @@ gradle.projectsEvaluated {
     subprojects.findAll { it.pluginManager.hasPlugin('java') }.forEach {
       testReportAggregation it
     }
-    subprojects.findAll { it.pluginManager.hasPlugin('jacoco') }.forEach {
-     jacocoAggregation it
+    // TODO: Fix build-tools code coverage task, now fails with:
+    // > The consumer was configured to find a component, preferably in the form of class files. 
+    //   However we cannot choose between the following variants of project :build-tools:
+    subprojects.findAll { it.pluginManager.hasPlugin('jacoco') && !it.name.equals("build-tools") }.forEach {
```

Comment:
The Jacoco aggregation task started to fail for `build-tools` with `...we cannot choose between the following variants of project :build-tools:`, caused by presence of the `'java-test-fixtures'` plugin. I was not able to find the way to fix the problem and have not yet able to trace the change that caused it.

Reply:
:partying_face: seems like found the solution ...